runtime/tests/integration_tests.rs

```diff
@@ -139,6 +140,109 @@ fn test_verify_cmd() {
     assert!(resp.is_none());
     assert_eq!(model.soc_ifc().cptra_fw_error_non_fatal().read(), 0);
 
+    #[allow(dead_code)]
+    #[derive(Debug)]
+    struct WycheproofResults {
+        id: usize,
+        comment: String,
+    }
+    // Collect all the errors and print at the end
+    let mut wyche_err: Vec<WycheproofResults> = Vec::new();
+    let test_set =
+        wycheproof::ecdsa::TestSet::load(wycheproof::ecdsa::TestName::EcdsaSecp384r1Sha384)
+            .unwrap();
+    // The mailbox has a fixed len for x and y so filter out untestable cases
+    let test_groups = test_set
+        .test_groups
+        .iter()
+        .filter(|x| x.key.affine_x.as_slice().len() <= 48)
+        .filter(|x| x.key.affine_y.as_slice().len() <= 48);
+    for test_group in test_groups {
+        // Since the mailbox uses R, S as input for signature use only tests with a valid signature
+        let tests = test_group
+            .tests
+            .iter()
+            .filter(|x| Signature::from_der(&x.sig).ok().is_some());
+        for test in tests {
+            model
+                .compute_sha512_acc_digest(test.msg.as_slice(), ShaAccMode::Sha384Stream)
+                .unwrap();
+
+            let cmd = EcdsaVerifyCmd {
+                chksum: 0,
+                pub_key_x: test_group.key.affine_x.as_slice()[..].try_into().unwrap(),
+                pub_key_y: test_group.key.affine_y.as_slice()[..].try_into().unwrap(),
+                signature_r: Signature::from_der(&test.sig)
+                    .unwrap()
+                    .r()
+                    .to_bytes()
+                    .into(),
+                signature_s: Signature::from_der(&test.sig)
+                    .unwrap()
+                    .s()
+                    .to_bytes()
+                    .into(),
+                // Do tests on mailbox
+            };
+            let checksum = caliptra_common::checksum::calc_checksum(
+                u32::from(CommandId::ECDSA384_VERIFY),
+                &cmd.as_bytes()[4..],
+            );
+            let cmd = EcdsaVerifyCmd {
+                chksum: checksum,
+                ..cmd
+            };
+            fn push_err(err: &mut Vec<WycheproofResults>, id: usize, comment: &String) {
+                err.push(WycheproofResults {
+                    id: id,
+                    comment: comment.clone(),
+                });
+            }
+            match model.mailbox_execute(u32::from(CommandId::ECDSA384_VERIFY), cmd.as_bytes()) {
+                Ok(resp) => match test.result {
+                    wycheproof::TestResult::Valid | wycheproof::TestResult::Acceptable => {
+                        if !resp.is_none() || model.soc_ifc().cptra_fw_error_non_fatal().read() != 0
+                        {
+                            push_err(&mut wyche_err, test.tc_id, &test.comment);
```

Comment:
If there's an error, we need to clear it after noting that.

Reply:
Thanks!

Reply:
Done